test/surface_test.py

```diff
@@ -741,8 +741,65 @@ def todo_test_blit(self):
         #
         # Pixel alphas will be ignored when blitting to an 8 bit Surface.
         # special_flags new in pygame 1.8.
-
-        self.fail()
+        def reset_dest(dst_surface):
+            """Resets our blank canvas destination surface"""
+            dst_surface.fill(pygame.Color(0,0,0))
+        
+        #Init
+        src_surface = pygame.Surface((256, 256), 32)
+        src_surface.fill(pygame.Color(255,255,255))
+        dst_surface = pygame.Surface((64, 64), 32)
+        reset_dest(dst_surface)
+
+        # Full coverage w/ overflow, specified with Coordinate
```

Comment:
How about putting all of these tests into a class with separate methods?

- makes it easier to run an individual method in development
- it's possible to see if more than one test fails

Reply:
Went ahead and created a proper class for the blit tests as suggested. I didn't want my setUp() polluting/slowing down the following unrelated tests, so I created GeneralSurfaceTests class to pick up the following tests. I think the other tests could also be segmented somewhat like TestSurfaceBlit, but it's out of scope for now. Let me know what you think.